### PR TITLE
Gives the toy emag a description showing that it is a toy.

### PR DESCRIPTION
**Pull request by MrDoomBringer**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

Keeps the toy emag consistent with other toys in the game. Players shouldn't be forced to risk damaging equipment to test if an emag is real or not.

:cl: MrDoomBringer
tweak: Due to DonkCo's recent budget cuts, their line of toy emags are now noticably less realistic. As such, you can now examine an emag to determine if it is a toy or not.
/:cl:

@improvedname was doing something like this but he closed his pr and I think deleted his branch so I guess I'll give it a go.

### CODE REVIEW

**Review by ShizCalev — DISMISSED** [review]

File: code/game/objects/items/cards_ids.dm [hunk 1]
```diff
@@ -78,7 +78,7 @@
 	A.emag_act(user)
 
 /obj/item/card/emagfake
-	desc = "It's a card with a magnetic strip attached to some circuitry."
+	desc = "It's a card with a magnetic strip attached to some circuitry. Closer inspection shows that this card is a poorly made replica, with a "DonkCo" logo stamped on the back."
```
Comment (SpaceManiac) [inline code/game/objects/items/cards_ids.dm]: Doesn't compile, the inner `"` need to be escaped.

Comment (MrDoomBringer) [reply code/game/objects/items/cards_ids.dm]: fuck webeditor has me slippin

**Comment by DaxDupont** [comment]
You can literally smack ANYTHING with this to make it honk.

Including windows.

**Comment by MrDoomBringer** [comment]
When people try to see if something's fake, they examine it. It's like that for every toy in the game. It's not intuitive to see an emag you think is could be a toy, and immediately go "i will slap this against a wall to see if its real or not". 

The only reason someone would smack it against a wall to test it would be if they knew beforehand that toy emags honked, which isn't explained anywhere in-game.

**Comment by DaxDupont** [comment]
It's supposed to be metabreaker/trap for a reason.

The idea was that people would be tricked into thinking it's real.

**Comment by improvedname** [comment]
More like i got killed for winning arcade gane

**Comment by MrDoomBringer** [comment]
Unobservant people should be tricked into thinking it's real, not people who examine shit that they are given, etc. That's why every toy in the game has a description saying that it is a toy.

File: code/game/objects/items/cards_ids.dm [hunk 2]
```diff
@@ -78,7 +78,7 @@
 	A.emag_act(user)
 
 /obj/item/card/emagfake
-	desc = "It's a card with a magnetic strip attached to some circuitry."
+	desc = "It's a card with a magnetic strip attached to some circuitry. Closer inspection shows that this card is a poorly made replica, with a /"DonkCo/" logo stamped on the back."
```
Comment (ShizCalev) [inline code/game/objects/items/cards_ids.dm]: `\` not `/`

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by improvedname** [comment]
+people can use this fact as excuse just to kill freely even if they know because they can just tell the admins they didnt knew it was fake

File: code/game/objects/items/cards_ids.dm (hunk 2 above)
Comment (MrDoomBringer) [reply code/game/objects/items/cards_ids.dm]: FuCK PHONES ARE HARD

**Review by ShizCalev — APPROVED** [review]

**Comment by vuonojenmustaturska** [comment]
@improvedname If a sec guy opens your backpack and sees anything with an emag icon on it, they'll just quickly pocket it and then murder you. Only later will they found out they were pranked. There is no situation where a sec looks at your shiny emag and goes "OwO wats this" and then lets you go after the emag doesn't work. You can only see the names and descriptions of items when they're out of the bag.

**Comment by Cyberboss** [comment]
You can right/shift click examine no?

**Comment by improvedname** [comment]
you can but currently its just the same description like the antag object

**Comment by MrDoomBringer** [comment]
@improvedname cyberboss was talking about right/shift click examining while the thing is in the backpack

**Pull request merged**